TurkishFilter/sections/specific.txt

```diff
@@ -3332,7 +3332,7 @@ gidahatti.com##a[href^="https://tr.hach.com"]
 ||cdn.xix*.com/bnr^
 ||imagizer.imageshack.us/a/img922/8810/wHDdVD.gif
 evdesifa.com##.post .code-block
-hwp.com.tr,konupara.com,lafmacun.net,finanskocaeli.com##.code-block
+dilbilgisi.net,hwp.com.tr,konupara.com,lafmacun.net,finanskocaeli.com##.code-block
```

Comment:
also blocks useful element `.code-block-16`

let's use `dilbilgisi.net##.code-block:not(.code-block-16)`